.gitignore

```diff
@@ -8,6 +8,7 @@
 [Bb]uild/
 [Ll]ibrary/
 [Aa]pp/
+[Aa]pp.meta
```

Comment:
It's kinda weird we need to specifically add this .meta file to gitignore. Where does it come from?

Reply:
I have an App folder in my project for some test scripts I wanted to test specifically in the Assets folder.